esp/mlrs-wireless-bridge/mlrs-wireless-bridge.ino

```diff
@@ -57,10 +57,11 @@ List of supported modules, and board which need to be selected
 // (for the generic module also SERIAL_RXD and SERIAL_TXD need to be defined)
 //#define USE_SERIAL_INVERTED
 
-// Wifi Protocol 0 = TCP, 1 = UDP
-#define WIFI_PROTOCOL  1
+#define WIRELESS_PROTOCOL 1 // 0 = WiFi TCP, 1 = WiFi UDP, 2 = Bluetooth
 
-// Wifi credentials
+
+//**********************//
+//*** Wi-Fi settings ***//
```

Comment:
WiFi not Wi-Fi
we use the former througout :)

Reply:
Done.